data/panel.css

```diff
@@ -415,13 +426,17 @@ input[type=range]::-moz-focus-outer {
 
 .progress {
     box-sizing: border-box;
-
     height: 50px;
     padding-top: 6px;
     width: 100%;
     cursor: pointer;
 }
 
+.minimized .progress {
+	height: 6px;
```

Comment:
Looks like there are some spacing issues in this file, would you mind checking that your editor is set to only use spaces over tabs and ensure that the tab spacing matches the rest of the file (4 spaces)